src/sql/workbench/parts/dashboard/widgets/properties/propertiesWidget.component.html

```diff
@@ -8,13 +8,19 @@
 	<div [style.margin-right.px]="_clipped ? 30 : 0" [style.width]="_clipped ? 94 + '%' : '100%'" style="overflow: hidden">
 		<span #child style="white-space : nowrap; width: fit-content">
 			<ng-template ngFor let-item [ngForOf]="properties">
-				<span style="margin-left: 10px; display: inline-block;">
+				<span *ngIf="!endpoints" style="margin-left: 10px; display: inline-block;">
```

Comment:
ah, oh, the changes here doesn't look right, the coupling between ADS core and an extension is too tight, I would suggest to update this component and make it generic that any one else can use.